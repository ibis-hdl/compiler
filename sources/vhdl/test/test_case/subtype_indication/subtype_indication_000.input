-- type_mark only
type_mark 

