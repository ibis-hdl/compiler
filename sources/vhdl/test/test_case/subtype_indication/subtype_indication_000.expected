(subtype_indication
  (type_mark<v>
    (name<v>
      (identifier
        type_mark
      )
    )
  )
)
